solutions/usaco-1181.mdx

```diff
@@ -0,0 +1,97 @@
+---
+id: usaco-1181
+source: USACO Bronze 2022 January
+title: Drought
+author: Chongtian Ma
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_prob3_bronze_jan22.html)
+
+## Explanation
+
+We can solve this problem greedily using one linear sweep and making prefixes of cows have equal hunger as we sweep from left to right. Thus, we can break this
```

Comment:
i don't think the bronze ppl would know half of what this means

Reply:
i changed the word "sweep" to "loop", is it better now lol?

Reply:
hm
how about
"Let's go through the cows from left to right, making sure that all the cows we've processed have the same hunger level."

Reply:
yeah that's better, thanks.